### ISSUE TITLE
[Accessibility] The Toot button when composing a toot is not accessible.

### ISSUE THREAD

**Issue by MarcoZehe** [DagAgren/toot#125]
To reproduce:

1. Turn on VoiceOver (or watch the video I am going to post to my account later), and start a new toot.
2. Explore the top of the screen, where the heading is. Notice that either exploring, or swiping through the items in sequential order, will skip the Toot! button.

The reason is probably that this is a custom button that doesn't expose any accessibility information yet. The following needs to be changed at a minimum to make this accessible:

- IsAccessibilityElement: Yes.
- AccessibilityLabel: Toot! or whatever is appropriate.
- AccessibilityTraits: AccessibilityTraitButton.
- AccessibilityFrame: If the visual indicator VoiceOver shows does not correspond to the toot button, its screen coordinates and dimensions (relative to screen, not app) must be specified.
- AccessibilityActivate (optional) to perform the action if for some reason the default doesn't work. By default, VoiceOver tries to simulate a tap in the center of the item.

After fixing it, the Toot! button should be accessible and reachable via VoiceOver.

**Comment by DagAgren** [comment]
After a lot of work I have managed to implement a lot of this functionality on most cell types. Not quite everything is there but I think it will do for a first attempt.
